src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8221,4 +8221,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_MemberNameSameAsExtendedType" xml:space="preserve">
     <value>'{0}': extension member names cannot be the same as their extended type</value>
   </data>
+  <data name="ERR_UnsupportedFeatureInRuntimeAsync" xml:space="preserve">
+    <value>Method '{0}' uses a feature that is not supported by runtime async currently. Opt the method out of runtime async by attributing it with 'System.Runtime.CompilerServices.RuntimeAsyncMethodGenerationAttribute(false)'.</value>
```

Comment:
>currently [](http://example.com/codeflow?start=78&length=9)

Is the idea that this message will be removed before shipping the feature?
If yes, let's mark the usages with follow-up comments.
If no, let's remove "currently" (ie. we'll specify that this is not supported).

Reply:
I'm going to leave currently for now. It will eventually be removed (arglist will never be supported), but I don't think it's worth adding another message.